app/src/main/java/com/jetbrains/handson/mpp/mobile/MainActivity.kt

```diff
@@ -156,9 +169,19 @@ class MainActivity : AppCompatActivity(), ApplicationContract.View {
             if (isDeparture) {
                 presenter.setDepartureStation(adapter.getItem(position))
                 departureName = adapter.getItem(position).toString()
+                context.closeKeyboard()
```

Comment:
no need to write this twice, can do if (isDeparture) { ... } else { ... } context.closeKeyboard()